### ISSUE TITLE
Does this use the new HTTP/2 protocol (and hence we can increase payload to 4k) or not?

### ISSUE THREAD

**Issue by janngobble** [aufflick/p5-net-apns-persistent#5]
https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/CreatingtheNotificationPayload.html#//apple_ref/doc/uid/TP40008194-CH10-SW1

Apple says the new payload size is 4k  if you use http/2 w/json instead of the APNs binary interface.  Is this the way this works or not? I have to currently split up data I am sending to the app and use 2 pushes. 1 push would be far far better.

Thanks!

**Comment by aufflick** [comment]
No, this uses the original protocol only. It's not something I've used for a while - perhaps you may be able to convince @thunter to join in upgrading it :)
